docs/DeveloperGuide.md

```diff
@@ -133,6 +125,70 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### \[Implemented\] Find by specific fields feature
+
+The user can find assignments by name, module code, due date or time or priority level of assignment.
```

Comment:
Slight nitpick, why not change from "or" to "and/or" to specify you can use multiple keywords!

Reply:
multiple keywords are only allowed for the same type of field though! I will amend later to see how i can btr phrase it :)